src/core/thread/mle_router.cpp

```diff
@@ -2678,7 +2668,22 @@ ThreadError MleRouter::SendChildUpdateRequest(Child *aChild)
     Ip6::Address destination;
     Message *message;
 
+    if ((aChild->GetDeviceMode() & ModeTlv::kModeRxOnWhenIdle) == 0)
+    {
+        uint8_t childIndex = mNetif.GetMle().GetChildIndex(*aChild);
+
+        // for sleepy child, not send again if there has already queued indirect child update request message
```

Comment:
Suggestion for comment: `// No need to send "Child Update Request" to the child if there is one already queued for it.`